apis/nucleus/src/components/listbox/ListBoxSearch.jsx

```diff
@@ -8,9 +8,15 @@ import InstanceContext from '../../contexts/InstanceContext';
 
 const useStyles = makeStyles((theme) => ({
   root: {
+    border: `1px solid ${theme.palette.divider}`,
```

Comment:
When looking at the styling it looks like the fieldset element is used as the level where the borders is applied. Can these styles applied on the root level be moved to the fieldset?

Reply:
Yes, I could try moving these styles onto the `fieldset` instead. That may be more "correct" actually.

Reply:
Indeed that looks better as we get the blue highlighting on focus 👍

Reply:
Good! I tested it with analytic creation and it looks good in there as well.